hr_holidays_natural_period/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+For using natural period on leaves:
+
+#. Go to *Leaves > Dashboard*.
+#. Select dragging on the calendar the days you want to be on leave, or go
+   to the form view for selecting start and end dates.
+#. Select the proper "Leave Type" that has "Natural day" selected in "Request unit".
```

Comment:
Testing functionally I found a little UI problem: If you change dates, the amount of days change, but if you change days, the dates don't adjust, so you could get whatever amount of days for whatever range of dates

Reply:
It's interesting UI case but, i had checked similar in runbot 13.0 with other type and it work equal (change duration days NOT re-calculate dates).

Reply:
Mmm, intersting and confusing. Does it make any sense?

Reply:
IMO it's a bug because it's need to auto-adjust date end (according to number of days) but maybe it's a question to odoo, right? @pedrobaeza , what do you think about it?

Reply:
Yeah, I'm thinking in an Odoo issue :slightly_smiling_face: 

The easy fix to prevent this would be to make the field readonly in view anyway

Reply:
That's something that shouldn't be done in this module though, so let's not block it here.